cookbooks/ey-postgresql/templates/default/postgresql.conf.erb

```diff
@@ -191,7 +191,13 @@ archive_timeout = <%= @archive_timeout %>		# force a logfile segment switch afte
 
 max_wal_senders = 5		# max number of walsender processes
 #wal_sender_delay = 200ms	# walsender cycle time, 1-10000 milliseconds
+
+# as per chnage in postgres version 13 `wal_keep_segments` is renamed to `wal_keep_size` https://pgpedia.info/w/wal_keep_segments.html
+<% if @postgres_version >= "13.0" %>
+wal_keep_size = 128 # in logfile segments, 16MB each; 0 disables 
```

Comment:
This doesn't look right to me. It's not just a rename, the values are different too

https://www.postgresql.org/docs/13/release-13.html

> Rename configuration parameter wal_keep_segments to [wal_keep_size](https://www.postgresql.org/docs/13/runtime-config-replication.html#GUC-WAL-KEEP-SIZE) (Fujii Masao)
> 
> This determines how much WAL to retain for standby servers. It is specified in megabytes, rather than number of files as with the old parameter. If you previously used wal_keep_segments, the following formula will give you an approximately equivalent setting:
> 
> wal_keep_size = wal_keep_segments * wal_segment_size (typically 16MB)

For PG < 13 we had:

 ```wal_keep_segments = 128```

For PG >= 13 with the same WAL retention we need wal_keep_size = 128 * 16MB (default, I don't see this overridden in the conf file) = 2048MB:

```wal_keep_size = 2048```